app/assets/stylesheets/application.css

```diff
@@ -53,6 +53,31 @@ a {
   align-self: center;
 }
 
+.log-out {
```

Comment:
- [ ] I am getting an error when trying to run your project coming from this path Blog-App-II/app/assets/config/manifest.js
![image](https://user-images.githubusercontent.com/62705947/193831701-ca0341ed-e622-4086-9f6a-460e71c5f1f1.png)

Reply:
- [ ] Your tests are failing, kindly update them
![image](https://user-images.githubusercontent.com/62705947/193831987-cf115388-7949-4c7b-94a9-776b122a3922.png)
![image](https://user-images.githubusercontent.com/62705947/193832534-ff5df844-e6a3-4ef5-ad04-3b2a51ec6a29.png)

Reply:
### All issues are fixed now! Thank you.